[2022-09-19 23:43:45,989] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: my_dag.task1 manual__2022-09-19T18:13:43.593163+00:00 [queued]>
[2022-09-19 23:43:45,996] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: my_dag.task1 manual__2022-09-19T18:13:43.593163+00:00 [queued]>
[2022-09-19 23:43:45,996] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-09-19 23:43:45,996] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-09-19 23:43:45,996] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-09-19 23:43:46,017] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): task1> on 2022-09-19 18:13:43.593163+00:00
[2022-09-19 23:43:46,020] {standard_task_runner.py:52} INFO - Started process 1676 to run task
[2022-09-19 23:43:46,022] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'my_dag', 'task1', 'manual__2022-09-19T18:13:43.593163+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpd3zgk_wp', '--error-file', '/tmp/tmprne_43ad']
[2022-09-19 23:43:46,023] {standard_task_runner.py:80} INFO - Job 8: Subtask task1
[2022-09-19 23:43:46,049] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.task1 manual__2022-09-19T18:13:43.593163+00:00 [running]> on host 8eb0791cd07b
[2022-09-19 23:43:46,081] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=task1
AIRFLOW_CTX_EXECUTION_DATE=2022-09-19T18:13:43.593163+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-19T18:13:43.593163+00:00
[2022-09-19 23:43:46,082] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-19 23:43:46,082] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'hello world'"]
[2022-09-19 23:43:46,088] {subprocess.py:85} INFO - Output:
[2022-09-19 23:43:46,089] {subprocess.py:89} INFO - hello world
[2022-09-19 23:43:46,089] {subprocess.py:93} INFO - Command exited with return code 0
[2022-09-19 23:43:46,112] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=task1, execution_date=20220919T181343, start_date=20220919T181345, end_date=20220919T181346
[2022-09-19 23:43:46,153] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-19 23:43:46,171] {dagrun.py:630} WARNING - Failed to get task '<TaskInstance: my_dag.train_model_A manual__2022-09-19T18:13:43.593163+00:00 [removed]>' for dag 'my_dag'. Marking it as removed.
[2022-09-19 23:43:46,172] {dagrun.py:630} WARNING - Failed to get task '<TaskInstance: my_dag.train_model_B manual__2022-09-19T18:13:43.593163+00:00 [removed]>' for dag 'my_dag'. Marking it as removed.
[2022-09-19 23:43:46,172] {dagrun.py:630} WARNING - Failed to get task '<TaskInstance: my_dag.train_model_C manual__2022-09-19T18:13:43.593163+00:00 [removed]>' for dag 'my_dag'. Marking it as removed.
[2022-09-19 23:43:46,173] {dagrun.py:630} WARNING - Failed to get task '<TaskInstance: my_dag.choose_best_model manual__2022-09-19T18:13:43.593163+00:00 [removed]>' for dag 'my_dag'. Marking it as removed.
[2022-09-19 23:43:46,173] {dagrun.py:630} WARNING - Failed to get task '<TaskInstance: my_dag.accurate manual__2022-09-19T18:13:43.593163+00:00 [removed]>' for dag 'my_dag'. Marking it as removed.
[2022-09-19 23:43:46,174] {dagrun.py:630} WARNING - Failed to get task '<TaskInstance: my_dag.inaccurate manual__2022-09-19T18:13:43.593163+00:00 [removed]>' for dag 'my_dag'. Marking it as removed.
[2022-09-19 23:43:46,179] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
